client/components/MessageCondensed.vue

```diff
@@ -118,6 +118,18 @@ export default defineComponent({
 									(obj[type] > 1 ? " modes were set" : " mode was set")
 							);
 							break;
+						case "away":
+							strings.push(
+								String(obj[type]) +
+									(obj[type] > 1 ? " times marked away" : " time marked away")
```

Comment:
not sure '5 times marked away' makes sense. How about `Marked away N times`?

Reply:
ack